crates/autopilot/src/database/quotes.rs

```diff
@@ -23,4 +23,8 @@ impl Maintaining for Postgres {
             .await
             .context("fee measurement maintenance error")
     }
+
+    fn name(&self) -> Maintainer {
```

Comment:
nit: I would actually make this a `&'static str`. My reasoning is that `Maintaining` implementations can also be defined by consumers of the `shared` crate, so conceptually, it feels weird that the dependency knows about implementation details of its dependents.

Reply:
Indeed. Initially I set it to be `&str` but then I was worried because `&str` does not enforce name to be unique for different components across the system. New `Maintaining` implementation would need to check other implementations to see which names are occupied.
I don't have a strong opinion here since both ways are not perfect.

Reply:
Switched back to `&str`